.circleci/config.yml

```diff
@@ -69,6 +69,8 @@ jobs:
           key: v4-npm-cache
           paths: "assets/node_modules"
 
+      - run: mix format --check-formatted
```

Comment:
This ensures we don't introduce changes that haven't been formatted into the codebase in the future